src/button-group/index.d.ts

```diff
@@ -3,6 +3,8 @@ import {StyledFn, StyletronComponent} from 'styletron-react';
 import {SHAPE, SIZE, KIND} from '../button';
 import {Override} from '../overrides';
 
+export {SIZE};
```

Comment:
question: why is this change (and the one in pagination) required?

Reply:
There is a conflict of SIZE constants within those files:
```
src/button-group/index.d.ts:3:16 - error TS2440: Import declaration conflicts with local declaration of 'SIZE'.

3 import {SHAPE, SIZE, KIND} from '../button';
```

I've removed the declaration of `SIZE` constant at the bottom of button-group/index.d.ts and pagination/index.d.ts files and I've reexported it from 'button' module ( no need to redeclare it really ).